### DISCUSSION TITLE
What's up with Tablo Tool

### DISCUSSION THREAD

**Discussion by dougK44883** [jessedp/tablo-tools-electron#100] [Q&A] [answered]
Can you pass along any info as to the development of Tablo Tools?  As-is, it's reasonably stable!  Does seem in the beginning, maybe it was the height of the pandemic and stay-at-home...maybe with regular development.

Not much for nearly a year...  The CLI had 1 alpha build, any chance this may become a reality?

issues / enhancements / bugs --  just wondering is Tablo Tools developed as far as it's going to be?  
 It's really great as it is now, not complaining, again, just asking for any update you can or care to provide.

**Comment by jessedp** [comment, marked as answer]
The cli didn't work well on Windows. My intent had been to try newer versions of Electron to see if it that helped. At the same time I've been more interested in "rewriting" it in typescript and fixing some of the plumbing before adding other features on. That's relatively stalled, though as can be seen in the [typescript-conversion branch](https://github.com/jessedp/tablo-tools-electron/tree/typescript-conversion), I have made considerable headway.

As it stands and you noted, it is relatively stable and no one has really made requests or come up with show stoppers (unrelated to pre-releases). So when/if I finish up the conversion there's a chance I'll revamp stuff just cause, take another stab at the cli, and maybe add some stuff that no has asked for. 

And, honestly, running into the non-network available nonsense they've done with the Tablo Dual HDMI in #96 didn't exactly give me warm fuzzies about them continuing to support the unofficial APIs/mostly open device access. You can make that correlation looking at the dates in that thread and the branch I mentioned.
